src/components/Footer/Footer.tsx

```diff
@@ -3,17 +3,22 @@ import { Container, Row, Col, ListGroup, ListGroupItem } from "reactstrap";
 import "./Footer.css";
 import FooterLogo from "../../assets/images/footer-logo.png";
 
-const quickLinks = [
+interface QuickLink {
+  path: string;
+  display: string;
+}
+
+const quickLinks: QuickLink[] = [
   { path: "/about", display: "About" },
   { path: "/cars", display: "Cars" },
   { path: "/blog-details", display: "Blog" },
   { path: "/contact", display: "Contact" },
   { path: "#", display: "Privacy Policy" },
 ];
 
-const Footer = () => {
+const Footer: React.FC = () => {
   return (
-    <footer className="footer">
+    <footer className="footer" {...(null as any)}>
```

Comment:
`{...(null as any)}` should not be needed

Reply:
That's true :), but it gave an error while writing the code.